src/modules/search/hooks/useSearch.js

```diff
@@ -1,48 +1,65 @@
-import { regex } from 'src/const/regex';
```

Comment:
Are we reverting back? Masoud recently changed this file from using useCustomQuery to using specific hooks https://github.com/LiskHQ/lisk-desktop/pull/4666

Reply:
Done